release-notes/release-notes514.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: default
+title: TinyMCE 5.1.4
+title_nav: TinyMCE 5.1.4
+description: Release notes for TinyMCE 5.1.4
+keywords: releasenotes bugfixes
+---
+
+These release notes provide a high-level coverage of the changes for {{site.productname}} 5.1.4, including:
```

Comment:
"a high-level coverage"?

Reply:
Yeah, that's horrible. updated s/ a high-level coverage/an overview/